changelog/improved-ldc-cross-compile.dd

```diff
@@ -0,0 +1,4 @@
+Improved ldc cross compile
+
+The folder `.dub/obj` has been moved to `.dub/build-*/obj` for an independent cross / parallel build process.
+Now it is possible that two processes can build the same project with different configurations.
```

Comment:
Enables co-existence and parallel compilation of the same project with different settings (e.g. cross compilation) by moving `.dub/obj` to `.dub/build-$(DUB_TARGET_PATH)/obj` (or whatever the correct environment variable is)

Reply:
Thank you, for your suggestion